docs/guide/install.md

```diff
@@ -37,3 +37,57 @@ $ npx slidev
 > ```bash
 > echo 'shamefully-flatten=true' >> .npmrc
 > ```
+
+## Install on Docker
+
+If you need a rapid way to run a presentation, you can use the prebuilt (or even build your own) [docker](https://hub.docker.com/r/stig124/slidev) image (maintained by [stig124](https://github.com/Stig124))
```

Comment:
Thanks for managing the migrations. Since we already have the repo with the readme, how about let's keep this section with a link to the repo so we don't need to maintain duplicated docs?

Reply:
And there is done